apps/projects/app/components/Card/Revoked.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { Button, EmptyStateCard, GU, Text, useTheme } from '@aragon/ui'
+import { EmptyWrapper } from '../Shared'
+import revoked from '../../assets/revoked.svg'
```

Comment:
Can we export a component, instead of raw SVG? Like we did for [`IconDoubleCheck` in Allocations](https://github.com/AutarkLabs/open-enterprise/pull/1853/files#diff-373c0e728dced2965861263f8f033827)